ted_sws/domain/model/transform.py

```diff
@@ -7,24 +7,64 @@
 
 """ """
 import abc
+from typing import List
 
+from ted_sws.domain.model import PropertyBaseModel
 
-class RuleSet(abc.ABC):
+
+class MappingSuiteComponent(PropertyBaseModel, abc.ABC):
+    class Config:
+        validate_assignment = True
+        orm_mode = True
```

Comment:
The special constructor `from_orm` must be used to create the model instance if `orm_mode = True`. 
Moreover, I do not think we deal with any ORM. We want to avoid one :) 

Take a look at this example. 
https://pydantic-docs.helpmanual.io/usage/models/#orm-mode-aka-arbitrary-class-instances